webplugin/js/app/mck-app.js

```diff
@@ -332,6 +330,8 @@ function ApplozicSidebox() {
         try {
             var options = applozic._globals;
             var widgetSettings = data.chatWidget || data.widgetTheme;
+            var sessionTimeout = options.sessionTimeout;
+            sessionTimeout == null && (sessionTimeout = widgetSettings && widgetSettings.sessionTimeout);
```

Comment:
@ab14bhardwaj priority should be given to the one passed from javascript

Reply:
@devashishmamgain Priority is given to appOption only.
**var sessionTimeout = options.sessionTimeout;** This line is setting the value passed from SessionTimeout.
If nothing is passed or the value is null or undefined then it will check that API is returning any response or not.
**sessionTimeout == null && (sessionTimeout = widgetSettings && widgetSettings.sessionTimeout);**

Reply:
got it :)